<launch>
  <node name="gps_conv" pkg="core_rover" type="utm_odometry_node">
	<remap from="odom" to="vo"/>
	<remap from="fix" to="/nova_common/gps_data"/>
	<param name="rot_covariance" value="99999"/>
	<param name="frame_id" value="base_footprint">
  </node>
</launch>
